dotcom-rendering/src/web/components/CalloutEmbed/FileUpload.tsx

```diff
@@ -0,0 +1,62 @@
+// File upload is the only legacy component that is still used on the old callout form
```

Comment:
This is the old `FileUpload` component. It's now going to live in the (legacy) `CalloutEmbed` folder as it is only used there.